Presentation.Web/Tests/Helpers/SideNavigation/ItSystemNavigation.ts

```diff
@@ -0,0 +1,19 @@
+﻿class ItSystemNavigation {
+
+    public static mainPage() {
+        element(by.css(`[data-ui-sref="${ItSystemNavigationSrefs.mainPageSref}"`)).click();
+    }
+
+    public static exposedInterfacesPage() {
+        element(by.css(`[data-ui-sref="${ItSystemNavigationSrefs.exposedInterfacesSref}"`)).click();
```

Comment:
Eventuelt smid by.css(`[data-ui-sref="${ItSystemNavigationSrefs.exposedInterfacesSref}"`) ned i i dens pageobject.

Reply:
Disse er en form for pageobjects i sig selv. Det er for at man kan navigere rundt på detail viewet på et ItSystem. Lige nu er der kun de sider jeg selv skal bruge, men de er lette at udvide og så er den funktionalitet samlet et sted